### PR TITLE
NPM script to update `@since n.e.x.t` tags

### LINKED ISSUE #90: Automate @since tags generation using the `n.e.x.t` placeholders
Currently, we use `1.0.0` for `@since` tags in the codebase. This is fine for now since we haven't released an actual version yet. Later on it may not work because some already implemented tickets might be postponed and included into later releases which means that we will need amend `@since` tags manually to match the release number.

What we can do is to start using `@since n.e.x.t` tags for new functions, methods, filters and actions instead of entering the exact release number. Then we can implement an npm script that will replace `n.e.x.t` with the correct version.

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Fixes #90

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

File: bin/plugin/commands/since.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	const patterns = [
+		path.resolve( __dirname, '../../../**/*.php' ),
+		path.resolve( __dirname, '../../../**/*.js' ),
+	];
+
+	const files = await glob( patterns, {
+		ignore: [ __filename, '**/node_modules', '**/vendor' ],
+	} );
+
+	const regexp = new RegExp( '@since n.e.x.t' );
```
Comment (felixarntz) [inline bin/plugin/commands/since.js]: Doesn't this need to have a global modifier? Otherwise it'll only replace one occurrence of this in each file.

**Review by felixarntz — COMMENTED** [review]
@eugene-manuilov Looks great, only one point of feedback.

File: bin/plugin/commands/since.js (hunk 1 above)
Comment (eugene-manuilov) [reply bin/plugin/commands/since.js]: Good catch. You are absolutely right, it should have the global modifier. Updated.

**Review by felixarntz — APPROVED** [review]
@eugene-manuilov Thanks, this looks great!

cc @JustinyAhin for the updated release instructions

**Review by JustinyAhin — APPROVED** [review]
This looks good to me too.
Thanks @eugene-manuilov

**Pull request merged**